README.md

```diff
@@ -36,7 +36,13 @@ This repo contains the sourcecode and ancillary information for Lancache.net and
 ## Building the site
 
 ### Jekyll & Github Pages
-This site is built using Jekyll and hosted on Github pages. To build at home for testing see [jekyllrb.com](https://jekyllrb.com/docs/)
+This site is built using Jekyll and hosted on Github pages. To build at home for testing :
+
+- Install Ruby according to the Jekyll docs [Jekyll Windows Install](https://jekyllrb.com/docs/installation/windows/)
```

Comment:
Why would we link directly to the windows instructions here? The general install root gives links to each OS which is why we chose it. Please revert this change

Reply:
My intention here was to try to reduce the friction required to get setup to be as low as possible, in the hopes that it will help encourage the community to contribute changes as much as possible.

I had no prior experience with Ruby or Jekyll, so I kept notes of the steps required to run Jekyll locally, and consolidated the steps onto the readme here.  The commands required (such as `bundle install`) are in their documentation, however I wanted to make no assumptions of people's experience level, and instead consolidate these steps front and center on this project's readme.

Regardless, I'll get this reverted and committed.